packages/pigeon/platform_tests/test_plugin/android/src/test/kotlin/com/example/test_plugin/AllDatatypesTest.kt

```diff
@@ -46,13 +53,16 @@ internal class AllDatatypesTest : TestCase() {
     assertTrue(firstTypes.aNullable4ByteArray.contentEquals(secondTypes.aNullable4ByteArray))
     assertTrue(firstTypes.aNullable8ByteArray.contentEquals(secondTypes.aNullable8ByteArray))
     assertTrue(firstTypes.aNullableFloatArray.contentEquals(secondTypes.aNullableFloatArray))
-    assertEquals(firstTypes.aNullableList, secondTypes.aNullableList)
-    assertEquals(firstTypes.aNullableMap, secondTypes.aNullableMap)
     assertEquals(firstTypes.nullableMapWithObject, secondTypes.nullableMapWithObject)
     assertEquals(firstTypes.aNullableObject, secondTypes.aNullableObject)
+    assertEquals(firstTypes.list, secondTypes.list)
+    assertEquals(firstTypes.boolList, secondTypes.boolList)
+    assertEquals(firstTypes.doubleList, secondTypes.doubleList)
+    assertEquals(firstTypes.intList, secondTypes.intList)
+    assertEquals(firstTypes.stringList, secondTypes.stringList)
+    assertEquals(firstTypes.map, secondTypes.map)
   }
 
-  @Test
```

Comment:
Why don't we want this annotation on tests?

Reply:
The class level annotation covers it.

Reply:
From what I see in JUnit docs, it looks like `extends TestCase` is a legacy mode, and the `@Test` annotation is the preferred way for newer code. Is the IDE giving different feedback?